.github/workflows/snyk-validator.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Snyk Validator'
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - '**/package-lock.json'
+      - '**/package.json'
+  pull_request:
+    branches: [master]
+    paths:
+      - '**/package-lock.json'
+      - '**/package.json'
+
+jobs:
+  renovate-config:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2
```

Comment:
Why not just `actions/checkout@v2` ? 

Same for node ?

Is that for security stuff ... ?
Because those are official Github actions, afterall.

Reply:
It's renovate managed in fact.
Renovate, when it runs, will resolve the tag and update the commit digest